sdk/src/test/java/io/opentelemetry/sdk/metrics/MeterSdkTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.metrics;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertSame;
+
+import com.google.common.collect.ImmutableMap;
+import io.opentelemetry.metrics.LongCounter;
+import java.util.Collections;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Unit tests for {@link MeterSdk}. */
+@RunWith(JUnit4.class)
+public class MeterSdkTest {
+
+  @Test
+  public void testLongCounter() {
+    MeterSdk testSdk = new MeterSdk();
+
+    LongCounter longCounter =
+        testSdk
+            .longCounterBuilder("testCounter")
+            .setConstantLabels(ImmutableMap.of("sk1", "sv1"))
+            .setLabelKeys(Collections.singletonList("sk1"))
+            .setDescription("My very own counter")
+            .setUnit("metric tonnes")
+            .setMonotonic(true)
+            .build();
+    assertNotNull(longCounter);
+    assertEquals(SdkLongCounter.class, longCounter.getClass());
+
+    // todo: verify that the MeterSdk has kept track of what has been created, once that's in place
+  }
+
+  @Test
+  public void testLabelSets() {
+    MeterSdk testSdk = new MeterSdk();
+
+    assertSame(testSdk.emptyLabelSet(), testSdk.emptyLabelSet());
+    assertEquals(testSdk.createLabelSet("key", "value"), testSdk.createLabelSet("key", "value"));
+    assertEquals(
```

Comment:
Currently we use truth for asserts in the rest of the code, consider to be consistent for the moment.

Reply:
If we want this to be a standard (and I honestly just prefer using the built-in asserts for the simple cases), we should write this up explicitly in our CONTRIBUTING.md doc.

But, ok, I can switch these over to use the more verbose approach.

Reply:
tests updated